content/go/notification_ocurrent_events.go.md

```diff
@@ -0,0 +1,235 @@
+---
+title: "Managing Notification Occurrence Events services"
+description: "How to interact with the Notification Ocurrence Events services. The SoftLayer_Notification_Occurrence_Event service represents all events with potential to cause a disruption in service.."
+date: "2020-10-07"
+classes: 
+    - "SoftLayer_Virtual_Guest"
+    - "SoftLayer_Metric_Tracking_Object"
+tags:
+    - "Metric Tracking Object "    
+---
+
+# Setup
+Each of these snippets below will share basically the same initialization code, so to save some space we will include the initialization code here, and assume you can setup the SoftLayer client before running each example.
+
+```Go
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/softlayer/softlayer-go/services"
+	"github.com/softlayer/softlayer-go/session"
+	"github.com/softlayer/softlayer-go/sl"
+)
+
+func main() {
+	username := "Set - me"
+	apikey := "Set - me"
+
+	// Create SoftLayer API session
+	sess := session.New(username, apikey)
+
+	notificationService := services.GetNotificationOccurrenceEventService(sess)
+
+	fmt.Println(getAllObjects(notificationService))
+	// Notificacion ocurrent event id = 123456
+    fmt.Println(getObject(notificationService,123456))
+}
+
+func getAcknowledgeNotification(service services.Notification_Occurrence_Event, notificationId int) bool {
+	service.Options.Id = sl.Int(notificationId)
+	result, err := service.AcknowledgeNotification()
+	if err != nil {
+		fmt.Printf("\n Unable to place order:\n - %s\n", err)
```

Comment:
"Unable to place order" isn't really a good error message here. Change all of these to something like "SoftLayer API Error"